src/server/socketHandlers/scGraphQLHandler.js

```diff
@@ -3,10 +3,9 @@ import Schema from 'server/graphql/rootSchema';
 
 // eslint-disable-next-line no-underscore-dangle
 const mutations = Schema._mutationType && Schema._mutationType._fields || {};
-const whitelist = ['updateUserProfile', 'summarizeMeeting'];
+const whitelist = ['createUserPicturePutUrl', 'updateUserProfile', 'summarizeMeeting'];
```

Comment:
it's the mutations that we want a response from. We want most mutations to mutate & then die because the new query will come back from the DB changefeed. These whitelisted ones have a response that we care about.